clusterman/kubernetes/kubernetes_cluster_connector.py

```diff
@@ -86,10 +87,18 @@ def get_resource_allocation(self, resource_name: str) -> float:
         )
 
     def get_resource_total(self, resource_name: str) -> float:
-        return sum(
+        # Total node resources minus any allocated resources to excluded pods
+        base_total = sum(
             getattr(total_node_resources(node), resource_name)
             for node in self._nodes_by_ip.values()
         )
+        excluded_total = sum(
+            getattr(allocated_node_resources(self._excluded_pods_by_ip[get_node_ip(node)]), resource_name)
+            for node in self._nodes_by_ip.values()
+        )
+        if excluded_total > 0:
+            logger.info(f'Excluded {excluded_total} {resource_name} from total resources')
```

Comment:
nit: I'd extend the info to say that the resources were excluded because they are being consumed by daemon sets to make it clear for debugging.

Reply:
Updated!

Output now shows:
```
INFO:clusterman.kubernetes.kubernetes_cluster_connector:Excluded 0.7000000000000001 cpus from daemonset pods
INFO:clusterman.kubernetes.kubernetes_cluster_connector:Excluded 1300.0 mem from daemonset pods
INFO:clusterman.kubernetes.kubernetes_cluster_connector:Excluded 100.0 disk from daemonset pods
INFO:clusterman.kubernetes.kubernetes_cluster_connector:Excluded 0 gpus from daemonset pods
```